readme.rst

```diff
@@ -23,6 +23,17 @@ It also will export the grid to Excel.
 For now, if you are interested in using it, you will need to see the source code and tests therein
 for example usage.
 
+Features
```

Comment:
Nice to see a docs update for this, even though there really aren't useful docs around it. This will get us started!